backend/api.py

```diff
@@ -12,6 +12,9 @@
 app = FastAPI()
 config = Config()
 
+if config.frontend_url is None:
```

Comment:
Why are we checking for just the `frontend_url` property? Also this is the only property I gave a default value